static/routing-tree.js

```diff
@@ -167,11 +168,13 @@ function massage(root) {
 
   root.matchers = matchers;
 
+  root.receiverConfig = getReceiverConfig(root.receiver, receivers);
```

Comment:
You updated this line, but you'll also need to update all the other references to those names.

Reply:
Indeed, sorry but this oversight. I've updated the code accordingly and also rebased on top of master. I've tested a few different scenarios locally and everything seem fine